Nodejs/Product/Nodejs/Templates/Files/TypeScriptJsConfig/jsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "exclude": [
+    "**/bin",
+    "**/bower_components",
+    "**/jspm_packages",
+    "**/node_modules",
+    "**/obj",
+    "**/platforms",
+    "**/wwwroot/lib"
```

Comment:
Correct; this section controls the initial list of input files and we will never consider them for any other behavior.

Reply:
Based on Ryan's input, any huge libraries should be added to the safelist instead of excluding the whole folder. I will remove this from the PR and analyze the libraries for the safelist.